src/coffee/run.coffee

```diff
@@ -203,6 +201,7 @@ module.exports = class
       .description 'Allows to publish, unpublish or delete (all) products of your SPHERE.IO project.'
       .option '--changeTo <publish,unpublish,delete>', 'publish unpublished products / unpublish published products / delete unpublished products'
       .option '--csv <file>', 'processes products defined in a CSV file by either "sku" or "id". Otherwise all products are processed.'
+      .option '-o, --output <file>', 'Path to the file the product\'s state will write the resulting output in only if the product is more than 500'
```

Comment:
I dont really understand this sentence, could we please rewrite it?

Reply:
That's just specified the output path where to write the result to.